subsys/logging/Kconfig

```diff
@@ -286,6 +286,62 @@ config LOG_BACKEND_UART
 	help
 	  When enabled backend is using UART to output logs.
 
+config LOG_BACKEND_FCB
+	bool "Enable flash backend"
+	depends on FCB
+	help
+	  When enabled backend is storing logs to flash.
+
+if LOG_BACKEND_FCB
+
+config LOG_BACKEND_FCB_FLASH_AREA_ID
```

Comment:
better us the label instead of the ID. You can get ID using the macro: `ID = FLASH_AREA_ID(label)`
The build system doesn't guarantee same ID among different builds.

Reply:
so: LOG_BACKEND_FCB_FLASH_AREA_LABEL maybe